docs/wallet/wallet-extension.md

```diff
@@ -24,104 +22,133 @@ Let's walk through the steps how to install and set up the Maiar DeFi Wallet ext
 
 ![maiar-defi-example-secretphrase](/wallet/wallet-extension/wallet_extension_step5.png)
 
-* For security purposes, you will be prompted to input some of the secret words.
+**Step 4:** Before proceeding to the next step, confirm that you have safely stored your secret phrase.
+
+**Step 5:** For further verification, you will be prompted to input some of the secret phrases. 
 
 ![maiar-defi-example-authentication](/wallet/wallet-extension/wallet_extension_step6.png)
 
-* Next, create a new password and confirm your password.
+**Step 6:** Create a password that will be used to access your wallet. Ensure you keep this password safe as it will be needed to access your wallet regularly and this password cannot be retrieved. 
 
 ![maiar-defi-example-password](/wallet/wallet-extension/wallet_extension_step7.png)
 
+**Step 7:** Completed! Your Maiar DeFi Wallet has been successfully created and set to be used. 
+
 ![maiar-defi-example-successful](/wallet/wallet-extension/wallet_extension_step8.png)
 
-**Completed successfully!**
 
+### Import existing wallet
 
-## Key features
+Do you already have a wallet?
 
-### Send to a wallet
+Then there is no need to create a new one. The Maiar Wallet Extension provides an option to import your existing wallet. However, to import an existing wallet, you must be able to access the **recovery (secret) phrase** of your wallet. 
 
-With this extension installed, you can send to a wallet from your web browser without having to enter your password, which is a key feature of the Maiar DeFi wallet extension.
+The Maiar wallet has a set of 24-secret phrases, which serve as your wallet’s key. Using your secret phrase to import an existing wallet does not affect your wallet, as all transaction history across all networks will be reflected in the wallet extension.
 
-![maiar-defi-example-transaction](/wallet/wallet-extension/wallet_extension_step9.png)
+**To get started:** 
 
-Click on "*Send*." Add the wallet address, amount, and data.
+**Step 1:** With the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm) installed. Click on ‘’Import existing wallet”.
 
-![maiar-defi-example-send](/wallet/wallet-extension/wallet_extension_step10.png)
+![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)
 
-### Deposit to wallet
+**Step 2:** Next, enter your 24-word secret phrase. You can either enter these phrases one at a time or you can simply paste in the phrase using the "_paste_" icon.
 
-To deposit into a wallet, click on "*deposit*" and share your QR code or wallet.
+![importwallet](https://user-images.githubusercontent.com/52820835/151404597-537e8639-e5e8-47a8-9b85-bcd4951bfd1e.PNG)
```

Comment:
Careful with these links. All images should be stored locally on our site, like: /wallet/wallet-extension/wallet_extension_step4.png